docs/best-practices/open-source-software-licenties.md

```diff
@@ -104,6 +104,29 @@ Andersom kan broncode ook openbaar worden gemaakt zonder OSS licentie. Daarmee b
 auteursrechtelijk beschermd en kan het dus niet worden hergebruikt, aangepast of worden gedeeld.
 Openbaarmaking geeft wel inzicht in de broncode.
 
+## Publieke Domein
+
+Zoals beschreven zorgt een open source licentie voor het opheffen van een groot deel van de
+beperkingen die voortvloeien uit het auteursrecht, in de vorm van een gebruikerslicentie. Een
+open source licentie verandert verder niks aan de aanwezigheid van het auteursrecht zelf. Die
+wordt er niet mee opgeheven.
+
+Door expliciet afstand te doen van al je rechten op het werk zorg je ervoor dat het werk toekomt
+aan het publieke domein. De broncode is op dat moment juridisch van niemand meer is, en daarmee
```

Comment:
- 2 x is in de zin ...
- het koppelwoord 'en' wordt meestal niet vooraf gegaan van een komma 😉